.circleci/config.yml

```diff
@@ -1,213 +1,81 @@
-# generator: n-circle2-cli
-# template: component
-
-references:
-
-  container_config_node: &container_config_node
-    working_directory: ~/project/build
-    docker:
-      - image: cimg/node:<< parameters.node-version >>
-    parameters:
-      node-version:
-        default: "18.16"
-        type: string
-
-  workspace_root: &workspace_root ~/project
-
-  attach_workspace: &attach_workspace
-    attach_workspace:
-      at: *workspace_root
-
-  npm_cache_keys: &npm_cache_keys
-    keys:
-      - v1-dependency-npm-{{ checksum "package.json" }}-
-      - v1-dependency-npm-{{ checksum "package.json" }}
-      - v1-dependency-npm-
-
-  cache_npm_cache: &cache_npm_cache
-    save_cache:
-      key: v1-dependency-npm-{{ checksum "package.json" }}-{{ epoch }}
-      paths:
-        - ./node_modules/
-
-  restore_npm_cache: &restore_npm_cache
-    restore_cache:
-      <<: *npm_cache_keys
-
-  filters_only_main: &filters_only_main
-    branches:
-      only: main
-
-  filters_ignore_main: &filters_ignore_main
-    branches:
-      ignore: main
-
-  filters_ignore_tags: &filters_ignore_tags
-    tags:
-      ignore: /.*/
-
-  filters_version_tag: &filters_version_tag
-    tags:
-      only:
-        - /^v?\d+\.\d+\.\d+(?:-beta\.\d+)?$/
-    branches:
-      ignore: /.*/
-
-filters_only_renovate_nori: &filters_only_renovate_nori
-  branches:
-    only: /(^renovate-.*|^nori/.*)/
-
-filters_ignore_tags_renovate_nori: &filters_ignore_tags_renovate_nori
-  tags:
-    ignore: /.*/
-  branches:
-    ignore: /(^renovate-.*|^nori/.*)/
-
+# CONFIG GENERATED BY DOTCOM-TOOL-KIT, DO NOT EDIT BY HAND
 version: 2.1
-
 orbs:
-  node: circleci/node@4.6.0
-
+  tool-kit: financial-times/dotcom-tool-kit@5
+executors:
+  node:
+    docker:
+      - image: cimg/node:16.14-browsers
```

Comment:
**chore:** We need to add the `@dotcom-tool-kit/circleci` option to set this to 18.4.